autoware_adapi_v1_msgs/vehicle/msg/Gear.msg

```diff
@@ -0,0 +1,9 @@
+# constants
+uint8 NONE = 0
```

Comment:
```suggestion
uint8 UNKNOWN = 0
```

What about setting default as unknown like the other messages? Or does `NONE` serve another functionality?

Reply:
I will change it to UNKNOWN

Reply:
changed at [4a60a79](https://github.com/autowarefoundation/autoware_adapi_msgs/pull/24/commits/4a60a79d378c93f9159a9d6dec0a0309fccdf7c9)